specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-09-02-preview/managedClusters.json

```diff
@@ -6529,6 +6533,16 @@
       },
       "description": "Node Restriction settings for the security profile."
     },
+    "ManagedClusterSecurityProfileCustomCATrustCertificates": {
+      "type": "array",
+      "items": {
+        "type": "string"
```

Comment:
I think you can add `format`: `byte` here, which will cause this to be rendered as a `[]byte` in clients. It doesn't change the actual serialization format (it's a base64 string), but it more correctly clarifies the expected format and allows clients to treat it as a byte array in the various languages that are generated.

We should probably use it here unless we have a compelling reason not to.